File: dotnet/PasswordGetter.cs
using System.Collections.Generic;
using System.IO;

namespace dotnet
{
    public class PasswordGetter
    {
        public IEnumerable<string> GetPasswordsHashes(string path)
        {
            var result = new List<string>();
            string line;

            FileStream fileStream = new FileStream(path, FileMode.Open);
            using (StreamReader reader = new StreamReader(fileStream))
            {
                while (!reader.EndOfStream)
                {
                    line = reader.ReadLine();
                    line = line.Trim();
                    if(line.Length == 64)
                    {
                        result.Add(line.ToLower());
                    }
                }
            }

            return result;
        }
    }
}

File: python/password_getter.py
def get_passwords_hashes(path:str) -> []:
    digests = []
    file = open(path, "r")
    for line in file:
        line = line.strip()
        if len(line) == 64: # 64 digits in sha 256
            digests.append(line.upper())
    
    print(len(digests))
    file.close()
    return digests